### PR TITLE
Bump fabric-sim to 1.0.8

### PR DESCRIPTION
**Pull request by pierventre**

### CODE REVIEW

File: fabric-sim/Chart.yaml [hunk 1]
```diff
@@ -7,8 +7,8 @@ name: fabric-sim
 description: ONOS Fabric Simulator service
 kubeVersion: ">=1.17.0"
 type: application
-version: 1.0.7
-appVersion: v1.0.4
+version: 1.0.8
+appVersion: v1.0.5
```
Comment (tomikazi) [inline fabric-sim/Chart.yaml:11]: bump to 1.0.6

Comment (pierventre) [reply fabric-sim/Chart.yaml]: done

**Review by tomikazi — APPROVED** [review]

**Comment by pierventre** [comment]
retest this please

**Comment by tomikazi** [comment]
retest this please

**Pull request merged**